src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let finalPrice = 0;
+  const fullPrice = days * 40;
+
+  if (days < 3) {
+    return fullPrice;
+  }
+
+  if (days < 7) {
+    finalPrice = fullPrice - 20;
+
+    return finalPrice;
+  }
+
+  if (days >= 7) {
+    finalPrice = fullPrice - 50;
+
+    return finalPrice;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, you should create variables for main values - 20, 50, 40, 3, 7